src/utils/stringFormat.tsx

```diff
@@ -213,6 +213,21 @@ export function generateTreeCareMessage(item: Activity): string {
   }
 }
 
+// convert emailer filter values to a text representation of the range
+export function formatActivityCountRange(
+  min: number | null,
+  max: number | null,
+  maxCount: number,
+): string {
+  if (min === null && max === null) {
```

Comment:
🔧 Is it still possible for the first `if` condition to be true?

Reply:
Nope - thanks for catching this!